src/main/java/com/leverx/employeestat/rest/repository/WorkRepository.java

```diff
@@ -16,4 +18,15 @@
     List<Work> findAll();
 
     void deleteById(WorkId workId);
+
+    @Query(nativeQuery = true, value =
+            "SELECT * FROM project_employee " +
+            "WHERE position_end_date < current_date OR position_start_date > current_date")
```

Comment:
probably bug here.
imagine the scenario that employee had an interview for the project in april and was agreed to start from the june. If we execute query in may we will incorectly treat it as active postion as the condition `position_start_date > current_date` is true. The correct filter will be (now >= startDate AND now < endDate)

Reply:
ah, sorry, everything correct here, I misunderstood the purpose on the method